Cargo.toml

```diff
@@ -102,7 +102,7 @@ enumset = "1.0.12"
 epaint = "0.22.0"
 glam = "0.22"
 gltf = "1.1"
-half = "2.3.1"
```

Comment:
This forces that feature on every crate in the workspace that uses `bytemuck`, with no way to opt-out.
Can this be specified just for the one or two crates that require it instead?